homeassistant/components/linkplay/strings.json

```
@@ -23,6 +23,13 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
   },
+  "selector": {
+    "sensor_select": {
+      "options": {
+        "use_ip_url": "Force IP in Home Assistant generated URLs to work around bugs in using local domain names with some devices."
+      }
+    }
+  },
```

Use the option flow translation section to let them show up. Something like this: 

```suggestion
"options": {
    "step": {
      "init": {
        "title": "LinkPlay options",
        "data": {
          "use_ip_url": "Force IP in Home Assistant generated URLs to work around bugs in using local domain names with some devices."
        }
      }
    }
  },
```